boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpunet_defconfig

```diff
@@ -16,11 +16,9 @@ CONFIG_PINCTRL=y
 # enable GPIO
 CONFIG_GPIO=y
 
-# IPM
-# HCI_RPMSG
-CONFIG_RPMSG_SERVICE=y
-CONFIG_RPMSG_SERVICE_MODE_REMOTE=y
+# Enable uart driver
+CONFIG_SERIAL=y
 
-CONFIG_HEAP_MEM_POOL_SIZE=8192
-
-CONFIG_BT=y
+# enable console
```

Comment:
Capital "E" for enable.
Since we don't expose the UART log from controller to public, would this part of the setting only for B0n?

Reply:
Yes, console only affect when b0n print boot banner.